Source/Engine/Level/Tags.cpp

```diff
@@ -55,6 +55,24 @@ Tag Tags::Get(const StringView& tagName)
     return tag;
 }
 
+Array<Tag> Tags::GetSubTags(Tag parentTag)
+{
+    Array<Tag> subTags = Array<Tag>();
```

Comment:
Why not just use `Array<Tag> subTags;` syntax?

Reply:
@Withaust fixed. Thanks